viam_cartographer.go

```diff
@@ -62,6 +62,7 @@ const (
 	SuccessMessage = "success"
 	// PostprocessToggleResponseKey is the key sent back for the toggle postprocess command.
 	PostprocessToggleResponseKey = "postprocessed"
+	editedMapName                = "/edited-map.pcd"
```

Comment:
Add a ticket to the Epic to call out that we're expecting the edited map packages to have this name from front end.